irs_lqr/irs_lqr.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np
+import time
+
+from irs_lqr.tv_lqr import TvLqr, get_solver
+
+class IrsLqrParameters:
```

Comment:
@pangtao22  this is the new parameter class. I think it might make sense to add a separate child of `IrsLqr` for the quasi-static sims after this!